test/px_blocking_messages/blocking.txt

```diff
@@ -3,162 +3,42 @@
-            <h1>Please verify you are a human</h1>
-        </div>
-    </div>
-    <div class="content-wrapper">
-        <div class="content">
-
-            <div id="px-captcha">
-            </div>
-            <p>
-                Access to this page has been denied because we believe you are using automation tools to browse the
-                website.
-            </p>
-            <p>
-                This may happen as a result of the following:
-            </p>
-            <ul>
-                <li>
-                    Javascript is disabled or blocked by an extension (ad blockers for example)
-                </li>
-                <li>
-                    Your browser does not support cookies
-                </li>
-            </ul>
-            <p>
-                Please make sure that Javascript and cookies are enabled on your browser and that you are not blocking
-                them from loading.
-            </p>
-            <p>
-                Reference ID: #8712cef7-bcfa-4bb6-ae99-868025e1908a
-            </p>
-        </div>
-    </div>
-    <div class="page-footer-wrapper">
-        <div class="page-footer">
-            <p>
-                Powered by
-                <a href="https://www.perimeterx.com/whywasiblocked">PerimeterX</a>
-                , Inc.
-            </p>
-        </div>
-    </div>
-</section>
-<!-- Px -->
-<script>
-    window._pxAppId = 'PXfake_app_ip';
-    window._pxJsClientSrc = '/fake_app_ip/init.js';
-    window._pxFirstPartyEnabled = true;
-    window._pxVid = 'bf619be8-94be-458a-b6b1-ee81f154c282';
-    window._pxUuid = '8712cef7-bcfa-4bb6-ae99-868025e1908a';
-    window._pxHostUrl = '/fake_app_ip/xhr';
-</script>
-<script>
-    var s = document.createElement('script');
-    s.src = '/fake_app_ip/captcha/captcha.js?a=&u=8712cef7-bcfa-4bb6-ae99-868025e1908a&v=bf619be8-94be-458a-b6b1-ee81f154c282&m=0';
-    var p = document.getElementsByTagName('head')[0];
-    p.insertBefore(s, null);
-    if (true) {
-        s.onerror = function () {
-            s = document.createElement('script');
-            var suffixIndex = '/fake_app_ip/captcha/captcha.js?a=&u=8712cef7-bcfa-4bb6-ae99-868025e1908a&v=bf619be8-94be-458a-b6b1-ee81f154c282&m=0'.indexOf('captcha.js');
-            var temperedBlockScript = '/fake_app_ip/captcha/captcha.js?a=&u=8712cef7-bcfa-4bb6-ae99-868025e1908a&v=bf619be8-94be-458a-b6b1-ee81f154c282&m=0'.substring(suffixIndex);
-            s.src = '//captcha.px-cdn.net/PXfake_app_ip/' + temperedBlockScript;
-            p.parentNode.insertBefore(s, p);
+    <script>
+        window._pxVid = '{{vid}}';
+        window._pxUuid = '{{uuid}}';
+        window._pxAppId = '{{appId}}';
+        window._pxHostUrl = '{{hostUrl}}';
+        window._pxCustomLogo = '{{customLogo}}';
+        window._pxJsClientSrc = '{{jsClientSrc}}';
+        window._pxFirstPartyEnabled = {{firstPartyEnabled}};
+        var script = document.createElement('script');
+        script.src = '{{blockScript}}';
```

Comment:
This is for the tests -- the values should be filled in with those from the test